lib/ember-cli/asset_resolver.rb

```diff
@@ -0,0 +1,53 @@
+module EmberCLI
+  class AssetResolver
+    def initialize(options)
```

Comment:
Unfortunately, there are Rails 3.1 users we'd like to support.

Am I too pessimistic assuming that if users are tied up with Rails 3.1 their Ruby versions won't be up to date?

Our [support matrix](https://github.com/thoughtbot/ember-cli-rails/blob/master/.travis.yml#L8-L11) tests against 2.0 -> 2.2, even though that commit is only a few days old.

What are your opinions on library support for older rubies?